packages/api-sdk/src/index.test.ts

```diff
@@ -1,13 +1,14 @@
 /* eslint-disable dot-notation */
 import JWT from 'jsonwebtoken';
 import * as s from 'superstruct';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
 
 import ApiSDK, { SGenerateClientParams, SParams } from '.';
 import { Client } from './client';
 import { PublicClient } from './publicClient';
 
 const createSDK = () => {
-  const assert = jest.spyOn(s, 'assert');
+  const assert = vi.spyOn(s, 'assert');
```

Comment:
this looks like a package switch (rather than a formatting or update), could i get a description/reasoning of `vi` over `jest`?

Reply:
1. we've already moved to `vitest` in most other repos, at this point there are only a few holdouts (`platform`)
2. it's generally faster than `jest` or `mocha` (we were using `swc` to try to combat this with `jest`)
3. it's self-contained (we don't need additional libraries like `chai` or `sinon` like we do with `mocha`
4. it plays well with `vite` which we're using for our frontend, which saves us from having to re-invent our build configuration whenever we want to test (it can be a real pain with all the [config needed for a mature SPA](https://github.com/voiceflow/creator-app/blob/master/apps/creator-app/config/vite/config.ts))
5. it has almost the same API as `jest` so shouldn't require a significant mental switch